surface/include/pcl/surface/impl/marching_cubes.hpp

```diff
@@ -233,36 +242,12 @@ pcl::MarchingCubes<PointNT>::performReconstruction (pcl::PolygonMesh &output)
     return;
   }
 
-  // Create grid
-  grid_ = std::vector<float> (res_x_*res_y_*res_z_, 0.0f);
-
-  // Populate tree
-  tree_->setInputCloud (input_);
-
-  getBoundingBox ();
-
-  // Transform the point cloud into a voxel grid
-  // This needs to be implemented in a child class
-  voxelizeData ();
-
+  // real part of marching cube
+  performReconstructionProc();
```

Comment:
See my comment below regarding the signature change for this block and the code that can also be included inside.